pages/docs/concepts/asyncapi-document/reply-info.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Adding reply Info
+weight: 210
+---
+
+Request/Reply is a common messaging or communication pattern where one component or service, known as the requestor, sends a request to another component or service, known as the replier and waits for a response. This pattern is often used when a component needs to initiate an action and receive a specific response in return either synchronously or asynchronously. This communication pattern is typically asynchronous, meaning that the requester does not block and wait for an immediate response. Instead, it can continue processing other tasks or even send out other requests while waiting for the reply to arrive. 
+
+In case of multiple requests made, each request is processed independently and sends the reply/response to the corresponding requestor when ready.
+
+The request/reply pattern in AsyncAPI works in the same fashion while supporting all the different sub-patterns. Irrespective of the sub-pattern you would like to implement, the Request/reply pattern can be implemented in the asyncAPI document using actions in the `Operation` object.
+
+You can add the reply info using the `Operation Reply` object under the `Operation` object. The `reply` object represents the response message.
+
+Here is diagram to illustrate the working of a basic request/reply pattern:
+```mermaid
+sequenceDiagram
+    participant Requester as Requester
+    participant Responder as Responder
+    Requester->>Responder: Send Request
+    activate Responder
+    Responder-->>Requester: Send Reply
+    deactivate Responder
+```
+
+## Sub-patterns
```

Comment:
did you get this from some external resources, this phrase like sub-pattern? As for me there are no such sub-patterns. It is just more about different way you can implement req/reply, when you know the message and its location by default, or it is dynamic

so I suggest not to use it and just have `dynamic response` section directly. And before you can have just a paragraph that in AsyncAPI you can represent req/reply it 2 different ways, when "requester specifies in in runtime in request where response should be sent" and "when requester know exactly where the response will be sent"